### PR TITLE
CSI-3736 bump base images

### PR DESCRIPTION
**Pull request by oriyarde**

### CODE REVIEW

File: Dockerfile-csi-controller [hunk 1]
```diff
@@ -29,7 +29,7 @@ RUN ./csi_pb2.sh
 RUN pip3 install .
 
 
-FROM registry.access.redhat.com/ubi8/python-38:1-77
+FROM registry.access.redhat.com/ubi8/python-38:1-80
```
Comment (ArbelNathan) [inline Dockerfile-csi-controller:32]: what about the `FROM` [above](https://github.com/IBM/ibm-block-csi-driver/blob/16ad98f5c311925648a2bcb3464b8db7cc0c8b33/Dockerfile-csi-controller#L15)?

Comment (ArbelNathan) [reply Dockerfile-csi-controller:32]: same in .test

Comment (oriyarde) [reply Dockerfile-csi-controller:32]: not relevant since https://github.com/IBM/ibm-block-csi-driver/pull/438

Comment (ArbelNathan) [reply Dockerfile-csi-controller:32]: it looks there that both images in the file have been downgraded, why is the second one is ok to bump?

Comment (oriyarde) [reply Dockerfile-csi-controller:32]: because it doesn't fail on compilation, it just contains the complied files

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Pull request merged**